src/Configuration/GrumPHP.php

```diff
@@ -41,6 +41,14 @@ public function getGitDir()
     }
 
     /**
+     * @return \Gitonomy\Git\Repository
+     */
+    public function getGitRepository()
```

Comment:
It's better to inject the repository directly in the task constructor instead of using the configuration as additional service container. Can you remove this method?